doc/user/content/ops/optimization.md

```diff
@@ -20,26 +20,71 @@ Building an efficient index for distinct **clauses** and **operators** can be pu
 `ORDER BY` and `LIMIT` aren't clauses that benefit from an index.
 
 ### `WHERE`
-Speed up a query involving a `WHERE` clause with equality comparisons, using the following table as a guide:
+Speed up a query involving a `WHERE` clause with equality comparisons to literals, using the following table as a guide. (`collection` can be a source, view, materialized
+view, or table. `$1`, `$2`, etc. mean literals, e.g., `42`, or `'foo'`.)
 
-Clause                    | Index                                                                   |
---------------------------|-------------------------------------------------------------------------|
-`WHERE x = $1`            | `CREATE INDEX ON view_name (x);`                                        |
-`WHERE x IN ($1)`         | `CREATE INDEX ON view_name (x);`                                        |
-`WHERE x * 2 = $1`        | `CREATE INDEX ON view_name (x * 2);`                                    |
-`WHERE upper(x) = $1`     | `CREATE INDEX ON view_name (upper(x));`                                 |
-`WHERE x = $1 AND y = $2` | `CREATE INDEX ON view_name (x, y);`                                     |
-`WHERE x = $1 OR y = $2`  | `CREATE INDEX ON view_name (x);`<br /> `CREATE INDEX ON view_name (y);` |
+Clause                    | Index                                |
+--------------------------|--------------------------------------|
+`WHERE x = $1`            | `CREATE INDEX ON collection (x);`    |
+`WHERE x IN ($1, $2, $3)` | `CREATE INDEX ON collection (x);`        |
+`WHERE (x, y) IN (($1, $2), ($3, $4))` | `CREATE INDEX ON collection (x, y);`     |
+`WHERE x = $1 AND y = $2` | `CREATE INDEX ON collection (x, y);`     |
+`WHERE (x = $1 AND y = $2) OR (x = $3 AND y = $4`) | `CREATE INDEX ON collection (x, y);`     |
+`WHERE x * 2 = $1`        | `CREATE INDEX ON collection (x * 2);`    |
+`WHERE upper(x) = $1`     | `CREATE INDEX ON collection (upper(x));` |
 
-**Note:** to speed up a query using a multi-column index, as in `WHERE x = $1 AND y = $2`, the query must use all the fields in the index chained together via the `AND` operator.
+You can verify that the index is being used for a lookup with `EXPLAIN`, which should show `lookup`:
```

Comment:
What about turning this excellent suggestion into an active voice?

Examples:
```suggestion
You can verify an index usage in a lookup using `EXPLAIN`. Search for `lookup` after the index name to confirm its usage. 
```

```suggestion
Use `EXPLAIN` to verify that a lookup query uses an index. Look for `ReadExistingIndex`/`lookup` to assert an index's usage:
```

Reply:
Thanks, I rewrote the sentences!